tests/test_fixtures_settings_factory.py

```diff
@@ -1,27 +0,0 @@
-"""
```

Comment:
Should we re-add one or two of these tests - as a kind of documentation/example of the fixtures?

Reply:
EDIT: I just found them.